prog/pwm/fancontrol.8

```diff
@@ -1,15 +1,20 @@
-.TH FANCONTROL 8 "September 2009" "lm-sensors 3"
+.TH FANCONTROL 8 "January 2018" "lm-sensors 3"
 .SH NAME
-fancontrol \- automated software based fan speed regulation
+fancontrol \- automated software based fan speed regulation to lower fan noise
 
 .SH SYNOPSIS
 .B fancontrol
 .I [configfile]
 
 .SH DESCRIPTION
+The purpose of \fBfancontrol\fP is to lower the noise level of the cooling fans
```

Comment:
Not really to "lower the noise level". That could be accomplished by setting all fans to their minimum speed or by turning them off. You go into this below but the explanation is a bit complicated. Something with optimize and tradeoff, maybe. I'd leave this change out for now unless we can find a really simple explanation.